huaweicloud/services/rds/data_source_huaweicloud_rds_engine.go

```diff
@@ -0,0 +1,109 @@
+package rds
+
+import (
+	"context"
+	"sort"
+
+	"github.com/chnsz/golangsdk/openstack/rds/v3/instances"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/helper/hashcode"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
+)
+
+var (
+	// Database type of MySQL
+	DBMySQL = "MySQL"
+	// Database type of Postgre SQL
+	DBPostgreSQL = "PostgreSQL"
+	// Database type of Microsoft SQL Server
+	DBSQLServer = "SQLServer"
```

Comment:
how about just using the raw value MySQL、PostgreSQL、SQLServer?

Reply:
Seems better, if we use them in other resources, the `DB` prefix of `rds.DBMySQL` seems to be a redundant string.